x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/filtering/FilteringRuleCondition.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.application.connector.filtering;
+
+public enum FilteringRuleCondition {
+    CONTAINS("contains"),
+    ENDS_WITH("ends_with"),
+    EQUALS("equals"),
+    GT(">"),
+    LT("<"),
```

Comment:
Nit: For no other reason than to use words on everything:

```suggestion
    GT("gt"),
    LT("lt"),
```

Reply:
Our protocol actually expects the `"<"` and `">"` in the doc so this is a workaround to support it. See reference [code in kibana](https://github.com/elastic/kibana/blob/main/packages/kbn-search-connectors/types/connectors.ts#L120)

Reply:
Thanks for explaining!